### PR TITLE
Copter: Add DDS details to release notes for 4.5

### PR DESCRIPTION
**Pull request by Ryanf55**
Start the release notes for DDS for 4.5. Let me know which branch they should target, and if I should copy them to other vehicles. 

THere is vehicle-specific support in DDS, so I might want to handle that. For example, the waypoint support only works in plane right now.

### CODE REVIEW

**Comment by rmackay9** [comment]
Hi @Ryanf55,

Thanks very much for this.  Against master is great (we will backport the change to the Copter and Plane branches).  If the changes could also be added to the Rover/release-notes.txt and ArduPlane/ReleaseNotes.txt that would be perfect!

File: ArduCopter/ReleaseNotes.txt [hunk 1]
```diff
@@ -116,6 +116,16 @@ Changes from 4.4.4
 8) Parameters renamed
     - COMPASS_TYPEMASK renamed to COMPASS_DISBLMSK
 9) ROS2 / DDS support
+    - Added support for EProsima MicroXRCEDDS as a transport in SITL and hardware
+    - Added sensor data topic support such as NavSatStatus and BatteryState
+    - Added velocity control support in Copter
+    - Added a new AP_ExternalControl library for generic control support in SI units
+    - Added support for building ArduPilot with the colcon build system
```
Comment (srmainwaring) [inline ArduCopter/ReleaseNotes.txt:123]: Also:
- Added ROS 2 launch scripts for SITL, MAVProxy and micro-ROS agent
- Add pytests for DDS client and ROS 2 launch scripts and integrate into CI

**Review by srmainwaring — APPROVED** [review]
Looks good @Ryanf55. Could add additional info about the support for ros2 launch and integrated testing.

File: ArduCopter/ReleaseNotes.txt (hunk 1 above)
Comment (Ryanf55) [reply ArduCopter/ReleaseNotes.txt:123]: Added! Thanks!

**Comment by Ryanf55** [comment]
> Hi @Ryanf55,
> 
> Thanks very much for this. Against master is great (we will backport the change to the Copter and Plane branches). If the changes could also be added to the Rover/release-notes.txt and ArduPlane/ReleaseNotes.txt that would be perfect!

Done!

**Comment by Ryanf55** [comment]
Test on CubeOrange with PPP before backport to 4.5.

**Comment by rmackay9** [comment]
I guess, "Added high level goal interface for waypoints" doesn't apply to Rover?  I'm sure you know what you're doing so merging!

**Comment by Ryanf55** [comment]
> I guess, "Added high level goal interface for waypoints" doesn't apply to Rover? I'm sure you know what you're doing so merging!

Yea, AP_EXternalControl only implements Plane for that piece. Pending some more support from either the dev team or the community, we can flesh out the features to be part of all the vehicles. I personally don't have enough time right now to get comprehensive coverage for all vehicles for every control feature.

**Pull request merged**